test/integration/nse_dataplane_heal_test.go

```diff
@@ -15,6 +16,10 @@ import (
 func TestDataplaneHealLocal(t *testing.T) {
 	RegisterTestingT(t)
 
+	if !nsmd_test_utils.IsBrokeTestsEnabled() {
```

Comment:
This looks strange, we should fix if they are unstable instead of marking/skipping them.

Reply:
We have 30min timeout in master:
https://circleci.com/workflow-run/bc1ce04e-38f4-4c82-b45c-d15c93f833d8

I want to get to a point where we skip problematic tests and pass everything else.
The we cna have issues about whatever is skipped to be addressed separately.